google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -316,6 +316,73 @@ def _create_load_subparser(parser):
+  pipeline_parser.add_argument('-nc', '--nocache', help='Don\'t use previously cached results',
+                               action='store_true')
+
+  # load arguments
+  pipeline_parser.add_argument(
+      '-lm', '--load_mode', help='One of create (default), append or overwrite for loading data',
+      choices=['create', 'append', 'overwrite'], default='create')
+  pipeline_parser.add_argument('-lf', '--load_format', help='The format of the source file in GCS',
+                               choices=['json', 'csv'], default='csv')
+  pipeline_parser.add_argument('--skip',
+                               help='The number of head lines to skip during load; useful for CSV',
+                               type=int, default=0)
+  pipeline_parser.add_argument('-s', '--strict', help='Whether to reject bad values and jagged '
+                                                      'lines when loading', action='store_true')
+  pipeline_parser.add_argument(
+      '-ld', '--load_delimiter', default=',', help='The inter-field delimiter for CVS (default ,) '
+                                                   'in load file')
+  pipeline_parser.add_argument(
+      '-lq', '--load_quote', default='"', help='The quoted field delimiter for CVS (default ") in '
+                                               'the load file')
+  pipeline_parser.add_argument('-lp', '--load_path', help='The path URL of the GCS load file(s)')
+  pipeline_parser.add_argument('-lt', '--load_table', help='The destination bigquery table name '
+                                                           'for loading.')
+
+  # execute arguments
+  pipeline_parser.add_argument(
+      '-em', '--execute_mode', help='The table creation mode for execution', default='create',
+      choices=['create', 'append', 'overwrite'])
+  pipeline_parser.add_argument(
+      '-l', '--large', help='Whether to allow large results during execution', action='store_true')
+  pipeline_parser.add_argument('-eq', '--execute_query', help='The name of query for execution',
+                               required=True)
+  pipeline_parser.add_argument(
+      '-et', '--extract_table', help='Destination table name for the execution results')
+
+  # extract arguments
+  pipeline_parser.add_argument(
+      '-xf', '--extract_format', choices=['csv', 'json'], default='csv',
+      help='The format to use for the export for extraction.')
+  pipeline_parser.add_argument(
+      '-c', '--compress', action='store_true',
+      help='Whether or not to compress the data after extraction')
+  pipeline_parser.add_argument(
+      '-H', '--header', action='store_true',
+      help='Whether to include a header line (CSV only) in extracted file')
+  pipeline_parser.add_argument(
+      '-xd', '--extract_delimiter', default=',',
+      help='The field delimiter to use (CSV only) for extraction')
+  # TODO(rajivpb): Consider not having this, and defaulting to "SELECT * FROM <composed-table-name>"
```

Comment:
Yeah, I'm confused as to why you would need a second query statement. Seems like the point of the execution step is to create the output that you want, in a temporary table, and the extraction step is just telling how that output should be dumped.

Reply:
Removed in https://github.com/googledatalab/pydatalab/pull/494